internal/controller/services/monitoring/monitoring/operator-prometheusrules.tmpl.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.rhobs/v1
+kind: PrometheusRule
+metadata:
+  name: operator-prometheusrules
+  namespace: {{.Namespace}}
```

Comment:
I guess because we are specifying empty resourceSelector in MonitoringStack (i.e. all labels) and namespaceSelector is not defined at all (which means monitor resources just in the same namespace) and no label is present on these PrometheusRules, then the Prometheus instance will discover rules just in the monitoring namespace, right?